src/modules/site-v2/base/views/tools/genome_browser.py

```diff
@@ -44,36 +94,70 @@ def get_dataset_release_or_latest(release_version = None):
 def genome_browser(release_version=None, region="III:11746923-11750250", query=None):
   dataset_release = get_dataset_release_or_latest(release_version)
 
-  wormbase_version_override = request.args.get('wormbase_version', None)
-  if (wormbase_version_override is not None) and is_valid_wormbase_version(wormbase_version_override):
-    wormbase_version = wormbase_version_override.upper()
+  # Allow WB version to be overridden w URL variable
+  wormbase_version_override_str = request.args.get('wormbase_version', None)
+  if WormbaseVersion.validate(wormbase_version_override_str):
+    wormbase_version = WormbaseVersion(wormbase_version_override_str)
+
+  # Default to version 276
   else:
-    wormbase_version = dataset_release.wormbase_version    
-    # OVERRIDE wormbase_version  (default to 276 until 283 IGB data is available) 
-    wormbase_version = 'WS276'
-
-  
-  # dataset_release_prefix = f'//storage.googleapis.com/elegansvariation.org/releases'
-  dataset_release_prefix = f"//storage.googleapis.com/caendr-site-public-bucket/dataset_release/c_elegans"
-
-  track_url_prefix = f'//storage.googleapis.com/elegansvariation.org/browser_tracks'
-  # track_url_prefix = f'//storage.googleapis.com/caendr-site-public-bucket/dataset_release/c_elegans/{dataset_release.version}/browser_tracks'
-
-  bam_bai_url_prefix = f'//storage.googleapis.com/elegansvariation.org/bam'
-  
-  VARS = {'title': f"Genome Browser",
-          'DATASET_RELEASE': int(dataset_release.version),
-          'strain_listing': get_isotypes(),
-          'region': region,
-          'query': query,
-          'alt_parent_breadcrumb': {
-            "title": "Tools",
-            "url": url_for('tools.tools')
-          },
-          'wormbase_version': wormbase_version,
-          'release_version': int(dataset_release.version),
-          'track_url_prefix': track_url_prefix,
-          'bam_bai_url_prefix': bam_bai_url_prefix,
-          'dataset_release_prefix': dataset_release_prefix,
-          'fluid_container': True}
-  return render_template('tools/genome_browser/gbrowser.html', **VARS)
+    # wormbase_version = wormbase_version_override or dataset_release.wormbase_version
+    wormbase_version = WormbaseVersion('WS276')
```

Comment:
move hardcoded value `WS276` to either datastore or external configuration object.

Reply:
This value isn't actually needed; removed in 1d2373488ff9e1a7c1a9f576eda55acc48012c2d.